report/assets/styles.css

```diff
@@ -542,6 +549,8 @@
 
 .lh-audit--fail .lh-audit__display-text {
   color: var(--color-fail-secondary);
+  /* to remove color indicators - test */
+  /* color: gray; */
```

Comment:
Remove these?

Also I think there are some Lighthouse opportunity CSS classes we can probably drop at this point.